djangocms_versioning/datastructures.py

```diff
@@ -11,6 +14,25 @@ def __init__(self, content_model, grouper_field_name):
     def _get_grouper_field(self):
         return self.content_model._meta.get_field(self.grouper_field_name)
 
+    @cached_property
+    def version_proxy(self):
+        class Meta:
+            proxy = True
+        ProxyVersion = type(
+            self.grouper_field.remote_field.model.__name__ + 'Version',
+            (Version, ),
+            {
+                'Meta': Meta,
+                '__module__': __name__,
+                '_content_model': self.content_model,
+            },
+        )
+        return ProxyVersion
+
+    @property
+    def grouper_model(self):
```

Comment:
Not as important as this property is simpler, but I think it would be good to have a test for this as well.